src/components/Navbar.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const Navbar = () => (
+  <nav>
+    <h1>Math Magicians</h1>
+    <ul>
+      <li>
+        <NavLink to="/home">Home</NavLink>
```

Comment:
- [ ] [optional] @Bleu-Yves-Sopoude Great work on your app, the pages are well implemented, I would suggest you change the home router from `/home` to `/`, this will ensure your home page is not empty.